### PR TITLE
Upgrade rake task

### PR DESCRIPTION
**Pull request by shakil-fg**

### CODE REVIEW

**Comment by sujoydatta** [comment]
ok

**Comment by mdmahmudurrahman** [comment]
OK

File: lib/tasks/_shared.rake [hunk 1]
```diff
@@ -28,18 +28,13 @@ def process_category_task name
...
-        author: author, publish_date: publish_date, page: page, rating: rating
-      puts "#{obj.class.name} has been created!"
-      obj.attributes.each {|_name, value| puts "#{_name}: #{value}"}
-    rescue => exception
-      _dump_errors exception
-    end
-  else
-    puts "Error: Argument conflict"
+  begin
+    obj = Book.create! category_id: category_id, title: title,
```
Comment (dieunb) [inline lib/tasks/_shared.rake]: should user factory_girl to fake data

File: lib/tasks/user_book.rake [hunk 2]
```diff
@@ -1,18 +1,18 @@
...
     end
   end
 end
 
-def process_user_book_task user_id, book_id, favourite_status, study_status
+def process_user_book_task user_id, book_id, favourite, status
   begin
-    obj = Like.create! user_id: user_id, book_id: book_id,
-      favourite_status: favourite_status, study_status: study_status
+    obj = UserBook.create! user_id: user_id, book_id: book_id,
```
Comment (dieunb) [inline lib/tasks/user_book.rake]: should user factory_girl to fake data

**Comment by mdmahmudurrahman** [comment]
OK.

File: config/application.rb [hunk 3]
```diff
@@ -23,6 +23,10 @@ class Application < Rails::Application
     config.i18n.available_locales = [:en, :vn]
     config.i18n.default_locale = :en
 
+    config.generators do |g|
+      g.factory_girl dir: 'factories'
```
Comment (dieunb) [inline config/application.rb]: double quote

**Pull request merged**